iocage_lib/ioc_create.py

```diff
@@ -547,66 +549,16 @@ def create_install_packages(self, jail_uuid, location, config,
             if r and len(r.groups()) >= 3:
                 repo = r.group(3)
 
-        # Connectivity test courtesy David Cottlehuber off Google Group
-        # TODO: Use python's dns.resolver here?
-        srv_connect_cmd = ["drill", "-t", f"_http._tcp.{repo} SRV"]
-        dnssec_connect_cmd = ["drill", "-D", f"{repo}"]
-        dns_connect_cmd = ["drill", f"{repo}"]
```

Comment:
Why are we dropping all of the DNS checks within the jail?

Reply:
Unfortunately drill was acting like a noop. It never sets its exit code to anything other than 0 in my testing. So it was replaced with the host trying to resolve. This helps us figure out the same root issue

Reply:
# drill @9.9.9.99 google.coma
Error: error sending query: Could not send or receive, because of network error
# echo $?                    
1

Reply:
Not always:

```
root@freenas[~]# drill google.coma ; echo $?
;; ->>HEADER<<- opcode: QUERY, rcode: NXDOMAIN, id: 49543
;; flags: qr rd ra ; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 0 
;; QUESTION SECTION:
;; google.coma. IN      A

;; ANSWER SECTION:

;; AUTHORITY SECTION:

;; ADDITIONAL SECTION:

;; Query time: 30 msec
;; SERVER: 192.168.122.1
;; WHEN: Mon Oct  8 13:59:53 2018
;; MSG SIZE  rcvd: 29
0
```
It was very unreliable, that's why I dropped relying on it's return code.

Reply:
```
root@freenas[~]# drill @8.8.8.8 google.coma ; echo $?
;; ->>HEADER<<- opcode: QUERY, rcode: NXDOMAIN, id: 25306
;; flags: qr rd ra ; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 0 
;; QUESTION SECTION:
;; google.coma. IN      A

;; ANSWER SECTION:

;; AUTHORITY SECTION:
.       86363   IN      SOA     a.root-servers.net. nstld.verisign-grs.com. 2018100801 1800 900 604800 86400

;; ADDITIONAL SECTION:

;; Query time: 42 msec
;; SERVER: 8.8.8.8
;; WHEN: Mon Oct  8 14:00:47 2018
;; MSG SIZE  rcvd: 104
0
```
Even if wanting to use a different DNS resolver.

Reply:
Look up succeeded, there is nothing yo report as error. It will error out on network failures or bad dns server.

Reply:
There is no answer, which in our usecase should mean failure. This other test will find that as well, in addition to the others.

Reply:
There was an answer, the answer that there are no records for that domain. It behaves as desired.